lib/browser/api/extensions.js

```diff
@@ -603,6 +603,28 @@ var windowInfo = function (win, populateTabs) {
   }
 }
 
+ipcMain.on('chrome-windows-get', function (evt, responseId, windowId, getInfo) {
```

Comment:
`chrome.windows.get` is needed by the `cryptotoken` extension, thus I implemented it